src/pages/History.js

```diff
@@ -1,12 +1,85 @@
 import styled from "styled-components";
-export default function History() {
+export default function History({ historyItems, clearHistory }) {
   return (
-    <Div>
+    <Section>
       <h1>History Page</h1>
-    </Div>
+
+      {historyItems.length >= 1 && (
+        <div>
+          <h3>Lieber Kunde, danke für Dein Vertrauen!!</h3>
```

Comment:
Keep in sync with the successive numbering of headlines. You don't yet have an h2. Adjust all your headlines accordingly in this component.

Reply:
but i do not need h2 now. later i will also delete h1 because i will not need it. Does it neccesary to use h1,2,3,4 when i only need  to start with h4(if i need the size that h4 has)?

Reply:
done